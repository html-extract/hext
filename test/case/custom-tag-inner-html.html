<div>
  <yt-formatted-string>This is some content in a custom tag</yt-formatted-string>
  <self-closing-custom-tag />
</div>
