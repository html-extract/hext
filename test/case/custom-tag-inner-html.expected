{"x":"\n  <yt-formatted-string>This is some content in a custom tag</yt-formatted-string>\n  <self-closing-custom-tag>\n</self-closing-custom-tag>"}
